src/ui/public/index_patterns/route_setup/load_default.js

```diff
@@ -7,11 +7,15 @@ const notify = new Notifier({
   location: 'Index Patterns'
 });
 
+const NO_DEFAULT_INDEX_PATTERN_MSG = `
+In order to visualize and explore data in Kibana,
+you'll need to create an index pattern to retrieve data from Elasticsearch.
+`;
+
 // eslint-disable-next-line @elastic/kibana-custom/no-default-export
 export default function (opts) {
   opts = opts || {};
   const whenMissingRedirectTo = opts.whenMissingRedirectTo || null;
-  let defaultRequiredToasts = null;
```

Comment:
huh, looks like this was previously just over complicated, right?  Can't see any way this array was used before, so lgtm!